.circleci/config.yml

```diff
@@ -133,6 +67,9 @@ jobs:
             - v1-dependencies-{{ checksum "package.json" }}-{{ arch }}
             # fallback to using the latest cache if no exact match is found
             - v1-dependencies-
+      - run:
+          name: Build package
+          command: yarn build
```

Comment:
Remove `yarn ci:prep` from tests since it's not necessary, but added it back here